airbyte-cdk/bulk/core/base/src/main/resources/application.yaml

```diff
@@ -1,12 +0,0 @@
-airbyte:
-  file-transfer:
-    enabled: ${USE_FILE_TRANSFER:false}
-    staging-path: ${AIRBYTE_STAGING_DIRECTORY:/staging/files}
-  resources:
-    disk:
-      bytes: ${CONNECTOR_STORAGE_LIMIT_BYTES:5368709120} # 5GB
-  flush:
-    rate-ms: 900000 # 15 minutes
-    window-ms: 900000 # 15 minutes
-  destination:
-    record-batch-size-override: ${AIRBYTE_DESTINATION_RECORD_BATCH_SIZE_OVERRIDE:null}
```

Comment:
for future reference: if this should have existed at all, it should have been in core/load not core/base

Reply:
yup, this pr is moving it in there (and also namespacing all the other properties under `airbyte.destination`, which is why git doesn't recognize the file move >.>)